### ISSUE TITLE
Viewer errors in test suite under Python 3.x

### ISSUE THREAD

**Issue by JDWarner** [scikit-image/scikit-image#1802]
These appear related to qt5 or the Matplotlib qt5 backend, but only appear in Python 3.x. [See here](https://travis-ci.org/scikit-image/scikit-image/jobs/95082214).

Basic form of the error: 

``` python
  File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/viewers/core.py", line 120, in __init__
    self.fig, self.ax = figimage(image)
  File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/utils/core.py", line 188, in figimage
    fig, ax = new_plot(figsize=figsize, dpi=dpi)
  File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/utils/core.py", line 156, in new_plot
    canvas = new_canvas(**fig_kw)
  File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/utils/core.py", line 136, in new_canvas
    canvas = FigureCanvas(figure)
  File "/home/travis/build/scikit-image/scikit-image/skimage/viewer/utils/core.py", line 116, in __init__
    FigureCanvasQTAgg.__init__(self, self.fig)
  File "/home/travis/venv/lib/python3.4/site-packages/matplotlib/backends/backend_qt5agg.py", line 216, in __init__
    super(FigureCanvasQTAgg, self).__init__(figure=figure)
  File "/home/travis/venv/lib/python3.4/site-packages/matplotlib/backends/backend_qt5agg.py", line 65, in __init__
    super(FigureCanvasQTAggBase, self).__init__(figure=figure)
  File "/home/travis/venv/lib/python3.4/site-packages/matplotlib/backends/backend_qt5.py", line 239, in __init__
    super(FigureCanvasQT, self).__init__(figure=figure)
AttributeError: 'figure()' is not a Qt property or a signal
```

**Comment by JDWarner** [comment]
Updated link to Travis build: https://travis-ci.org/scikit-image/scikit-image/jobs/95082214

**Comment by blink1073** [comment]
It looks like a bad interaction with matplotlib 1.5, which was added to http://travis-wheels.scikit-image.org/ on 30 November.  I tried it on my setup (matplotlib 1.5.0, python 3.4, pyqt 4.11 on OSX), and it works.  Any thoughts on this @tacaswell?

**Comment by blink1073** [comment]
This poses an answer, but only for Ubuntu 14.04: http://askubuntu.com/q/661346.

**Comment by tacaswell** [comment]
Probably related to the changes in pyqt to use cooperative inheritance, unfortunately I don't have time to dig into this now.

What are the values of `rcParams['backend']` and `rcParams['backend.qt4']`?

attn @mfitzp

**Comment by blink1073** [comment]
Yep, that was it, we were using `Agg` instead of `Qt4Agg`, thanks @tacaswell, fixed in #1812.
